app/assets/stylesheets/views/_worldwide-organisation.scss

```diff
@@ -58,3 +58,20 @@
     }
   }
 }
+
+.corporate-information .sub_navigation {
```

Comment:
I don't really understand the spirit of the convention. I notice that all the other classes have the `worldwide-organisation` prefix. Should this corporate info class also have the `worldwide-organisation` prefix? But then it would be too long.

Reply:
Maybe a question for @patrickpatrickpatrick. I've never really used BEM or similar before!